lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/variables/semantic_color_scales.scss

```diff
@@ -10,7 +10,7 @@
     --color-#{$color}-lightest: var(--color-#{$color}-95);
     --color-#{$color}-lighter:  var(--color-#{$color}-80);
     --color-#{$color}-light:    var(--color-#{$color}-70);
-    --color-#{$color}-hover:     var(--color-#{$color}-50);
+    --color-#{$color}-hover:    var(--color-#{$color}-50);
```

Comment:
@Jeremy-Walton This one seems a bit limiting in practice. I wonder if we need a hover for each semantic color in the range? Something worth discussing anyway.

Reply:
Yeah, hover is extremely specific and not used outside of buttons.

Basically, `btn--primary` uses `--color-primary-base`, but for a hover effect, jumping to `--color-primary-light` is too drastic of a jump which is why this was added.

Instead of adding the special hover semantic name, I could just use the scale color in the hover css for buttons. That seems not great either but it does avoid adding a single-use semantic token

Current:
```
.btn--primary {
  @extend %btn;

  background: var(--color-primary-base);
  color: var(--color-on-primary-base);

  &:hover {
    background: var(--color-primary-hover);
  }
}
```

Proposed:
```
.btn--primary {
  @extend %btn;

  background: var(--color-primary-base);
  color: var(--color-on-primary-base);

  &:hover {
    background: var(--color-primary-50);
  }
}
```